### DISCUSSION TITLE
如果這學期可以重來，你會？

### DISCUSSION THREAD

**Discussion by bestbanana295** [1101-GS4719-C/Python-Community-GS4719-C#1528] [General]
又到了一個學期的尾聲，相信大家這學期生活中都發生了許多大大小小的事，其中有些事情希望可以再做得更好，或是可做初步一樣的選擇。如果這學期可以重來的話，你會怎麼做呢？ 想聽聽看大家的想法。 我的話是希望可以把Python再更熟悉一些，期末考的時候才不會這麼狼狽。

**Comment by lyien1021** [comment]
我覺得我應該會告訴我自己不要攬太多事在身上，因為想做的事太多，導致我常常焦頭爛額，沒辦法把每件事都做好其實讓我感覺蠻沮喪的

**Comment by larry40305** [comment]
我會告訴自己該退選的科目早點退選，不要為難自己。

**Comment by aa103967622** [comment]
能夠重來的話我會好好讀我的化學，讓我期末不至於被當，我會好好地處理我的學分數，讓他不至於到30學分，讓我不至於處理不來。

**Comment by MaxChi870** [comment]
能重來的話，希望能讓自己少選修一些課，多安排些課外的社團或團體活動，做更多不同的嘗試，也要合理安排時間的分配，不然期末會累得跟狗一樣。

**Comment by Chiuhsianglin** [comment]
我應該還是會告訴自己要抓好時間，很多時候我都會拖到最後一刻才把作業做完或是書太晚看，這都導致我很焦慮、作息不正常。希望我可以養成規劃時間的好習慣。

**Comment by ThanatosJun** [comment]
如果時間能夠重來，我希望回到選擇學校的那個時候，選擇我所愛的，而不是選擇對我有利的，這樣我應該會更快樂。但時間好像無法如此倒退，所以我只能展延我所學，並接觸我所愛的，連接我所能運用的資源來達成我的目標。

**Comment by Alex11392** [comment]
如果這學期能重來，我會選擇將這學期走過的失敗和挫折在走一次，畢竟對我而言這些經驗和記憶，是生命的成長過程中不可或缺的。

**Comment by lin157644** [comment]
如果這學期能重來，我會想好找到自己想做的目標，而非三心二意導致最終時間緊迫結果成果受時間所限無法達到預期。也應該好好做好每日的規劃，避免讓學業過度影響作息，健康是人生重要的本錢。
